ui/StatusQ/src/plugin.cpp

```diff
@@ -0,0 +1,40 @@
+#include <QtQml/QQmlExtensionPlugin>
+
+#include <QZXing.h>
+#include <qqmlsortfilterproxymodeltypes.h>
+
+#include "StatusQ/QClipboardProxy.h"
+#include "StatusQ/modelutilsinternal.h"
+#include "StatusQ/rxvalidator.h"
+#include "StatusQ/statussyntaxhighlighter.h"
+#include "StatusQ/statuswindow.h"
+
+class StatusQPlugin : public QQmlExtensionPlugin {
+    Q_OBJECT
+    Q_PLUGIN_METADATA(IID QQmlExtensionInterface_iid)
+public:
+    void registerTypes(const char* uri) override
+    {
+        Q_ASSERT(uri == QLatin1String("StatusQ"));
+
+        qmlRegisterType<StatusWindow>("StatusQ", 0, 1, "StatusWindow");
+        qmlRegisterType<StatusWindow>("StatusQ", 0, 1, "StatusWindow");
+        qmlRegisterType<StatusSyntaxHighlighter>("StatusQ", 0, 1, "StatusSyntaxHighlighter");
+        qmlRegisterType<RXValidator>("StatusQ", 0, 1, "RXValidator");
+
+        qmlRegisterSingletonType<QClipboardProxy>("StatusQ", 0, 1, "QClipboardProxy", &QClipboardProxy::qmlInstance);
+
+        qmlRegisterSingletonType<ModelUtilsInternal>(
+            "StatusQ.Internal", 0, 1, "ModelUtils", &ModelUtilsInternal::qmlInstance);
+
+        QZXing::registerQMLTypes();
+        qqsfpm::registerTypes();
+    }
+
+    void initializeEngine(QQmlEngine* engine, const char* uri) override
+    {
+        QQmlExtensionPlugin::initializeEngine(engine, uri);
+    }
```

Comment:
Here you could load the external resources with `QResource::registerResource()`

Reply:
I've replied to you here: https://github.com/status-im/status-desktop/issues/10218